scripts/mk/ephemeral.mk

```diff
@@ -79,7 +81,8 @@ ephemeral-login: .old-ephemeral-login ## Help in login to the ephemeral cluster
 
 .PHONY: .old-ephemeral-login
 .old-ephemeral-login:
-	xdg-open "https://oauth-openshift.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com/oauth/token/request"
+	@if [ "$(CLUSTER)" == "" ]; then echo "error:CLUSTER is empty; checkout your secret/private.mk file to setup this value correctly"; exit 1; fi
```

Comment:
IMO this check is not necessary. Line 10 sets a default value. The check only fails when `private.mk` has an empty definition, which should never happen.

Reply:
thanks mate! updated!